data/campaigns/tutorial03_seafaring.wmf/scripting/texts.lua

```diff
@@ -172,7 +172,7 @@ conclusion = {
       p(_[[You’ve lead the expedition to a successful end and founded a new colony. I’ve sent out some geologists – they already report that they’ve found some iron ore.]]) ..
       p(_[[In this scenario, you’ve learned everything about seafaring: how to build ports and ships and how to send out an expedition. Remember that expeditions are sometimes the fastest way to reach essential resources – and sometimes the only one.]]) ..
```

Comment:
If we change this from "in this scenario" to "So far" it would imply that this is not the end and sound better. Rest of the changes sound very good.